apps/src/code-studio/components/progress/course_progress.jsx

```diff
@@ -74,14 +78,12 @@ const CourseProgress = React.createClass({
 
     const hasLevelProgress = Object.keys(this.props.perLevelProgress).length > 0;
 
-    // Don't yet support PLC
-    const progressRedesign = !professionalLearningCourse &&
-      experiments.isEnabled('progressRedesign');
+    const progressRedesign = experiments.isEnabled('progressRedesign');
```

Comment:
With groups we can show plc scripts, but peer reviews wont work with this experiment is enabled.